inst/rmarkdown/templates/trb/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,97 @@
+
+# Features
+The template has a number of features that enable quick and painless manuscript authoring.
+
+## Title Page
+The standard \LaTeX `\maketitle` command is not very versatile, so we have replaced it with a `verb1titlepage` environment. This means that the writers will be required to manually enter spacings based on the number of contributors, but the current settings (12pt between authors, 36pt before, and 60pt after them) seems to work well. 
+
+Near the bottom of the title page, TRB requires a count of the manuscript's words, figures, and tables. This template generates these counts automatically. The figure and table counts are simply pulled from the \LaTeX\ counters using the \verb1totcount1 package. The word count feature is not as straight-forward, as it utilizes a call to the system command \verb1texcount1. Thus to compile the document writers must enable \verb#\write18# in their \verb1pdflatex1 call.
+
+In the newest version of this template, we added the total count automatically. The total count basically adds not only the word count, but also the equivalent count (250 words) for each table.  Note that starting from 2018, Figures no longer count toward total word counts.  However, each paper can have only up to 6 figures in total.  The total word count is implemented using a customized command \verb1\totalwordcount1.  Please see the original code for more information. The `rticles` version of this template does not do word count automatically.
+
+## Page Layout
+The document has 1 inch margins as required, with the author's names in the left
+heading and the page number in the right. The authors heading will need to be
+edited by the writers; automating this from the title page command is not
+currently possible. Paragraphs leading sections and subsections are not
+indented, while all subsequent paragraphs in that section are. Section types are
+defined as outlined by the old TRB Author's Guide.
+
+The document is single-spaced in 12 point Times font. Times New Roman is a proprietary font and is therefore not available by installation in open-source software. While the differences between Times variants are negligible, Times New Roman itself can be used in Mac OSX by compiling under \verb1xelatex1.
+
+
+```{r figure-example, fig.cap="Generic plot."}
+# Generate some sample data, then compute mean and standard deviation
+# in each group
+Speed <- cars$speed
+Distance <- cars$dist
+plot(Speed, Distance,
+     panel.first = lines(stats::lowess(Speed, Distance), lty = "dashed"),
+     pch = 0, cex = 1.2, col = "blue")
+```
+
+```{r table-example}
+knitr::kable(mtcars[1:5, 1:6], format = "latex", table.envir = "table", 
+             booktabs = TRUE, caption = "Example Table") 
+```
+
+
+
+Bibliography styles
+===================
+
+TRB still wants numbered, unsorted citations. Here are two sample references: [@Feynman1963118; @Dirac1953888].
+You can also cite authors, like @Feynman1963118.
+
```

Comment:
We could add more example of content like equation and custom blocks using Markdown syntax. 
This would allow to add the missing section from the template
What do you think ? 
```suggestion

# Equations

Intelligent driver model equations from wikipedia (<https://en.wikipedia.org/wiki/Intelligent_driver_model>) moved to the left using \verb1amsmath1 package with \verb1fleqn1 options.

\begin{linenomath}
  \begin{flalign}
    &\dot{x}_\alpha = \frac{\mathrm{d}x_\alpha}{\mathrm{d}t} = v_\alpha \\
    &\dot{v}_\alpha = \frac{\mathrm{d}v_\alpha}{\mathrm{d}t} = a\,\left( 1 - \left(\frac{v_\alpha}{v_0}\right)^\delta - \left(\frac{s^*(v_\alpha,\Delta v_\alpha)}{s_\alpha}\right)^2 \right)
  \end{flalign}
\end{linenomath}

[Custom blocks](https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html) can also be used to create LaTeX environment but keep a markdown syntax within it.

:::: {.linenomath latex=true}
$$
s^*(v_\alpha,\Delta v_\alpha) = s_0 + v_\alpha\,T + \frac{v_\alpha\,\Delta v_\alpha}{2\,\sqrt{a\,b}}
$$
:::

# Conclusion

This conclude the presentation of this template

# Acknowledgements

Thanks to Gregory Macfarlane who created this **rticles** template in 2021.

```